nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/blob/BlobProviderDescriptor.java

```diff
@@ -41,6 +41,14 @@
      */
     public static final String TRANSIENT = "transient";
 
+    /**
+     * Specific property corresponding to the user accessing the blob provider. This will be used to check if this
+     * user has actually the necessary rights to read or write blobs.
+     * 
+     * @since 10.2
+     */
+    public static final String NUXEO_USER_PROPERTY = "nuxeo.user";
```

Comment:
Let's be a bit more specific here, and call this `CREATE_FROM_KEY_USERS = "createFromKey.users"`. And also have `CREATE_FROM_KEY_GROUPS = "createFromKey.groups"`.
With a Javadoc like "A comma-separated list of users that can create blobs in this blob provider based only on a key" (same for groups).